webapp/src/app/components/leaderboard/leaderboard.component.ts

```diff
@@ -0,0 +1,153 @@
+import { ChangeDetectionStrategy, Component, computed, inject } from '@angular/core';
+import { injectQuery } from '@tanstack/angular-query-experimental';
+import { LeaderboardEntry, LeaderboardService } from 'app/core/modules/openapi';
+import { PullRequestApprovedIconComponent } from 'app/ui/icons/PullRequestApprovedIcon.component';
+import { PullRequestChangesRequestedIconComponent } from 'app/ui/icons/PullRequestChangesRequestedIcon.component';
+import { PullRequestCommentIconComponent } from 'app/ui/icons/PullRequestCommentIcon.component';
+import { TableBodyDirective } from 'app/ui/table/table-body.directive';
+import { TableCaptionDirective } from 'app/ui/table/table-caption.directive';
+import { TableCellDirective } from 'app/ui/table/table-cell.directive';
+import { TableFooterDirective } from 'app/ui/table/table-footer.directive';
+import { TableHeadDirective } from 'app/ui/table/table-head.directive';
+import { TableHeaderDirective } from 'app/ui/table/table-header.directive';
+import { TableRowDirective } from 'app/ui/table/table-row.directive';
+import { TableComponent } from 'app/ui/table/table.component';
+import { lastValueFrom } from 'rxjs';
+
+const defaultData: LeaderboardEntry[] = [
```

Comment:
Hmm, I think the way to do it would be mocking the service for the storybook story somehow.

Reply:
This data should be part of the story though

Reply:
We can probably just delete this data set as it should only be for testing purposes. And then only display the leaderboard if we are able to fetch data from the server. The story currently uses its own testing data set.


Would that solve the issue for you?

Reply:
I mean the story fails to fetch the data and then falls back to this data, either remove it here or move it to the story.